import ErrorException from '../src/lib/ErrorException';

describe('ErrorException test', () => {

	it('Verifica la excepci贸n retornada indicando nombre de la excepci贸n', () => {
		const exception = new ErrorException('mensaje', 'ERROREXCEPTION');
		expect(exception).toEqual({ message: 'mensaje', name: 'ERROREXCEPTION' });
	});


	it('Verifica la excepci贸n retornada cuando no se indica nombre de la excepci贸n', () => {
		const exception = new ErrorException('mensaje');
		expect(exception).toEqual({ message: 'mensaje', name: 'ErrorException' });
	});

});